packages/react-tinacms-github/README.md

```diff
@@ -0,0 +1,73 @@
+# react-tinacms-github
+
+## Implementation:
+
+Add the root TinacmsGithubProvider component to our main layout. In this case, we will use Github Auth.
+```ts
+// YourLayout.ts
+import { authenticate }  from '@tinacms/github-auth'
+import TinacmsGithubProvider from 'react-tinacms-github'
+const enterEditMode = () =>
+  fetch(`/api/preview`).then(() => {
+    window.location.href = window.location.pathname
+  })
+const exitEditMode = () => {
+  fetch(`/api/reset-preview`).then(() => {
+    window.location.reload()
+  })
+}
+const YourLayout = ({ Component, pageProps }) => {
+  return (<TinacmsGithubProvider
+      authenticate={() => authenticate('/api/create-github-access-token')}
+      enterEditMode={enterEditMode}
+      exitEditMode={exitEditMode}>
+      {...children}
+    </TinacmsGithubProvider>)
+}
+```
+
+Add error handling to our forms which prompt Github-specific action when errors occur (e.g a fork no longer exists).
+```ts
+// YourSiteForm.ts
+const YourSiteForm = ({ form, children }) => {
+  useGithubErrorListener(form)
```

Comment:
I don't think this needs to be addressed now, but I'm pretty sure we could solve this without the user having to add this to their site. 

We might be able to add something like this to the `TinacmsGithubProvider`:

```
useSubscribable(cms.forms)
let activeForm = cms.forms.all()[0] // Right now assume there's only one form active on the page.
useGithubErrorListener(activeForm)
```